include/unifex/thread_unsafe_event_loop.hpp

```diff
@@ -225,6 +230,11 @@ namespace _thread_unsafe_event_loop {
       return at_operation<std::remove_cvref_t<Receiver>>{
           (Receiver &&) r, dueTime_, loop_};
     }
+    template <typename Receiver>
+    at_operation<std::remove_cvref_t<Receiver>> connect(Receiver&& r) const& {
```

Comment:
Can we remove the rvalue connect() overload now that we have a const-lvalue overload?